packages/api-utils/src/automations.ts

```
@@ -231,23 +231,22 @@ const getPerValue = async (args: {
+      await replacePlaceHolders({
+        models,
+        subdomain,
+        getRelatedValue,
+        actionData: { config: value },
+        target,
+        isRelated: op1Type === 'string' ? true : false
+      })
+    ).config || '';
+
+  if (isExpression) {
+    updatedValue = eval(updatedValue.replaceAll(/{{.*?}}/g, '0'));
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)